libs/langchain/langchain/agents/format_scratchpad/tools.py

```diff
@@ -10,10 +10,10 @@
 
 from langchain.agents.output_parsers.tools import ToolAgentAction
 
+ToolMessagesFormatter = Callable[[Sequence[Tuple[AgentAction, str]]], List[BaseMessage]]
 
-def _create_tool_message(
-    agent_action: ToolAgentAction, observation: str
-) -> ToolMessage:
+
+def create_tool_message(agent_action: ToolAgentAction, observation: str) -> ToolMessage:
```

Comment:
why did you mark this as public?

Reply:
Before i think it's a good utility function for custom formatter, seems not necessary after think twice.